Preprocessing/tst_data.py

```diff
@@ -0,0 +1,1357 @@
+"""
```

Comment:
*tst_data.py* contains classes / functions used in the differents preprocessing scripts. there is for now only one script that uses it : *tst_preprocessing.py* but could change.

*tst* is because it's the code that is written for **TST** data
*data* because it's **data**

I'm open for an other suggestion.

Reply:
lowercase is to follow Python naming convention.